### PR TITLE
fix for sharding

### PR DESCRIPTION
**Pull request by darkdarkdragon**
**What does this pull request do? Explain your changes. (required)**
updates stream-info-service for sharding

**Specific updates (required)**
- updates /stream endpoint handlers to use `db.update` method
- add `db.add` method 
- uses `db.add` method in stream-info-service
- in stream-info-service look for object by id and by playbackId

**How did you test each of these updates (required)**
manually

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/api/src/app/stream-info/stream-info-app.ts [hunk 1]
```diff
@@ -158,52 +151,25 @@ class statusPoller {
       const manifest = status.Manifests[mid]
       countSegments(si, manifest)
       if (needUpdate) {
-        const streamKey = `stream/${mid}`
-        const storedInfo: Stream = await this.store.get(streamKey, false)
+        let storedInfo: Stream = await db.stream.get(mid)
+        if (!storedInfo) {
```
Comment (iameli) [inline packages/api/src/app/stream-info/stream-info-app.ts:155]: Not required now but I've been thinking a function on the stream table that's like `db.stream.getByIdOrPlaybackIdOrStreamKey` would be a good idea

File: packages/api/src/app/stream-info/stream-info-app.ts [hunk 2]
```diff
@@ -158,52 +151,25 @@ class statusPoller {
...
+          const [objs, _] = await db.stream.find({ playbackId: mid })
+          if (objs && Array.isArray(objs) && objs.length) {
```
Comment (iameli) [inline packages/api/src/app/stream-info/stream-info-app.ts]: This is a little defensive, `objs` will definitely always be an array

File: packages/api/src/app/stream-info/stream-info-app.ts [hunk 3]
```diff
@@ -253,8 +219,9 @@ export default async function makeApp(params) {
     broadcaster,
   } = params
   // Storage init
+  await db.start({ postgresUrl, postgresReplicaUrl: undefined })
```
Comment (iameli) [inline packages/api/src/app/stream-info/stream-info-app.ts:222]: Did this cause problems somewhere?

File: packages/api/src/store/table.ts [hunk 4]
```diff
@@ -143,6 +143,25 @@ export default class Table<T extends DBObject> {
     }
   }
 
+  async add(id: string, doc: T) {
```
Comment (iameli) [inline packages/api/src/store/table.ts:147]: Took me a long time to understand what was happening here. Looks good, but maybe let's add a comment:
```suggestion
  // Takes in an object of {"field": number} and increases all the fields by the specified amounts
  async add(id: string, doc: T) {
```

**Review by iameli — APPROVED** [review]
One documentation change you can just auto-apply but LGTM otherwise

File: packages/api/src/app/stream-info/stream-info-app.ts (hunk 3 above)
Comment (darkdarkdragon) [reply packages/api/src/app/stream-info/stream-info-app.ts:222]: No, why are you asking?

**Pull request merged**